ansible/module_utils/port_utils.py

```diff
@@ -130,6 +130,30 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
                     port_alias_to_name_map["Ethernet%d/%d" % (i, j)] = "Ethernet%d" % ((i - 1) * 8 + j - 1)
             port_alias_to_name_map["Ethernet65"] = "Ethernet512"
             port_alias_to_name_map["Ethernet66"] = "Ethernet513"
+        elif hwsku == "Arista-7060X6-64PE-C224O8":  # This hwsku uses every second OSFP port.
+            for i in range(1, 13, 2):
+                for j in range(1, 9):
+                    port_alias_to_name_map["Ethernet%d/%d" % (i, j)] = "Ethernet%d" % ((i - 1) * 8 + j - 1)
+            port_alias_to_name_map["Ethernet13/1"] = "Ethernet96"
+            port_alias_to_name_map["Ethernet13/5"] = "Ethernet100"
+            for j in range(1, 9):
+                port_alias_to_name_map["Ethernet14/%d" % (j)] = "Ethernet%d" % (14 * 8 + (j - 1))
```

Comment:
This should be port 15.

Reply:
can we replace the for loop with the following?

```
port_alias_to_name_map = {}
for i in range(1, 65, 2):
        if i in [13, 17, 45, 49]:
            for j in [1, 5]:
                port_alias_to_name_map["Ethernet%d/%d" % (i, j)] = "Ethernet%d" % ((i - 1) * 8 + j - 1)
        else:
            for j in range(1, 9):
                port_alias_to_name_map["Ethernet%d/%d" % (i, j)] = "Ethernet%d" % ((i - 1) * 8 + j - 1)

```

Reply:
Thanks @sdszhang, updated in latest commit.